src/setup/rgw.py

```diff
@@ -359,7 +359,7 @@ def _update_rgw_config_with_endpoints(conf: MappedConf, endpoints: dict, instanc
         # Update this with same config that is define for 1st instance.
         if instance == 1:
             radosgw_admin_log_file = os.path.join(
```

Comment:
@gargpalak048 with current structure, we are not creating directories for radosgw-admin log file.
if this directory is not present then this file will not be created.
to fix this could you please handle directory creation as well ?

Reply:
This validation is already done in _get_log_dir_path() 
No need to add the check again